src/System.ComponentModel.TypeConverter/dir.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.2.0.0</AssemblyVersion>
+  	<!-- don't change the Asssembly Version as one test would break -->
+    <AssemblyVersion>9.9.9.9</AssemblyVersion>
```

Comment:
We cannot change this assembly version here. This actually changes the produce version which we do not want to do. 

If we want to just set it for the test project you need to put the AssemblyVersion property directly in the test project.